### PR TITLE
cdr: for the council

### PR DESCRIPTION
**Pull request by icepuma**

### CODE REVIEW

File: cdr/CDR-001-council.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Communication channels - Should we use Discord? Should we use Matrix? Use GitHub Discussion?
+- Events - Another Hackathon?
+- Content - Community blog?
+- ...
+
+easier, a group of people (which includes [@rawkode](https://github.com/rawkode)) should discuss said topics and put them to a vote with the community.
+
+## Decision
+
+A council of 3 (to get a quorum) meets once a month to discuss the aforementioned points / topics. The council session happens in public and each decision should be written down as an ADR.
```
Comment (rawkode) [inline cdr/CDR-001-council.md:21]: Let's go with 5 or 7. I don't want this to be a burden on anyone and if they're unavailable, decisions should still be able to proceed.

Comment (icepuma) [reply cdr/CDR-001-council.md]: Updated to 5. We start with 5 and see how it goes :)

**Comment by rawkode** [comment]
@icepuma I made some changes to this. I don't think we need a council. I'd like anyone to be able to submit frictions and CDRs and to vote.

So let's lay that bedrock and take it from there?

**Comment by icepuma** [comment]
lgtm, merge it

**Pull request merged**